Assets/LeapMotionModules/UI/LeapMotionHandsExtentionAPI/ValueMappingExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿using UnityEngine;
+
+public static class ValueMappingExtensions {
+
+  /// <summary>
+  /// Maps the value between valueMin and valueMax to its linearly proportional equivalent between resultMin and resultMax.
+  /// </summary>
+  public static float Map(this float value, float valueMin, float valueMax, float resultMin, float resultMax) {
+    if (valueMin == valueMax) return resultMin;
+    return Mathf.Lerp(resultMin, resultMax, ((value - valueMin) / (valueMax - valueMin)));
+  }
+
+  public static Vector2 Map(this Vector2 value, float valueMin, float valueMax, float resultMin, float resultMax) {
+    return new Vector2(value.x.Map(valueMin, valueMax, resultMin, resultMax),
+                       value.y.Map(valueMin, valueMax, resultMin, resultMax));
+  }
+
+  public static Vector3 Map(this Vector3 value, float valueMin, float valueMax, float resultMin, float resultMax) {
+    return new Vector3(value.x.Map(valueMin, valueMax, resultMin, resultMax),
+                       value.y.Map(valueMin, valueMax, resultMin, resultMax),
+                       value.z.Map(valueMin, valueMax, resultMin, resultMax));
+  }
+
+  /// <summary>
+  /// Clamps the value between 0 and 1.
+  /// </summary>
+  public static float Clamp01(this float value) {
```

Comment:
Adding new functionality is one thing, making it so that you don't need to type Mathf. is a little more debatable :P  I would argue any existing functionality should not be re-routed through extension methods, but would probably be something good to talk about with the rest of the team too.